src/Uno.UWP/Storage/Pickers/FileSavePicker.wasm.cs

```diff
@@ -0,0 +1,50 @@
+#if __WASM__
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Runtime.InteropServices;
+using System.Text;
+using System.Threading.Tasks;
+using Uno.Extensions;
+using Uno.Foundation;
+using Windows.Foundation;
+using Windows.Storage.Provider;
+
+namespace Windows.Storage.Pickers
+{
+	public partial class FileSavePicker
+	{
+		public PickerLocationId SuggestedStartLocation { get; set; }
+
+		public IAsyncOperation<StorageFile> PickSaveFileAsync() => PickFilesTask().AsAsyncOperation();
+
+		public FileSavePicker()
+		{
+			SuggestedStartLocation = PickerLocationId.DocumentsLibrary;
+			FileTypeChoices = new Dictionary<string, IList<string>>();
+		}
+
+		private async Task<StorageFile> PickFilesTask()
+		{
+			if (!FileTypeChoices.Any())
+			{
+				throw new COMException();
```

Comment:
Even if Windows throws a `COMException`, we usually go with a more meaningful exception type, `InvalidOperationException` would be valid here.
```suggestion
				throw new InvalidOperationException();
```